src/pages/index.js

```diff
@@ -1,12 +1,14 @@
 import React from "react"
 import { Link } from "gatsby"
-
+import Navigation from "../components/Navigation"
 import Layout from "../components/layout"
 import SEO from "../components/seo"
 
 const IndexPage = () => (
-  <Layout>
-    <SEO title="Home" />
+  <>
+    <Navigation />
+    {/* <Layout> */}
+    {/* <SEO title="Home" /> */}
```

Comment:
⭐⭐⭐ _(must change)_ The SEO tag is what's creating opengraph stuff for us, so we don't want to comment it out.

```suggestion
    <SEO title="Home" />
```

Reply:
Solved  ✅